website_sale_cache/static/description/index.html

```diff
@@ -0,0 +1,67 @@
+<section class="oe_container">
+    <div class="oe_row oe_spaced">
+        <div class="oe_span12">
+            <h2 class="oe_slogan" style="color:#875A7B;">Website Sale Cache</h2>
+            <h3 class="oe_slogan">Use this module to greatly accelerate the loading of a page with a large number of product categories
+</h3>
+        </div>
+    </div>
+</section>
+
+<section class="oe_container">
+  <div class="oe_row oe_spaced">
+	<h3 class="oe_slogan">Usage</h3>
+        <div class="oe_span12" >
+          <p class="oe_mt32">
+			Enable the display of product categories on website by using the customize menu at the upper right corner.
+                        Once the categories are cached after first load, the page loading speed will be increased.
```

Comment:
Change it to "Open your website shop and load the page containing products. After first load the website product categories are cached and the page loading speed will be increased."